### PR TITLE
Fix not available label

### PR DESCRIPTION
**Pull request by andxu**
1. Fix the println error when process is exited
2. Use the disable for bad hint message: ' - '

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java [hunk 1]
```diff
@@ -394,7 +406,7 @@ private void addHighLight(final JComponent component) {
         synchronized (borderMap) {
             if (!borderMap.containsKey(component)) {
                 borderMap.put(component, component.getBorder());
-                component.setBorder(new LineBorder(Color.MAGENTA, 1));
+                component.setBorder(new LineBorder(Color.decode("0x8a2da5"), 1));
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java:409]: Make the color as a const with name

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java [hunk 2]
```diff
@@ -404,15 +416,13 @@ private void triggerPersistentStorage() {
 
         boolean enablePersist = StringUtils.equalsIgnoreCase(text, ENABLE_TEXT);
         if (enablePersist) {
-            Font font = publicUrlHyperLink.getFont();
             if (viewModel.isEnablePersistentStorage()) {
                 renderPersistent(this.viewModel);
             } else {
-                this.persistentLabel.setText("Persistent storage is not available before you save the settings.");
-                persistentLabel.setFont(new Font(font.getName(), Font.ITALIC, font.getSize()));
+                this.persistentLabel.setText("Persistent storage will be displayed here after you save the settings.");
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java:422]: displayed -> updated? As displayed may make feel storage path is available now but will not be displayed

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java]: Done

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java (hunk 1 above)
Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java]: done

**Pull request merged**